.github/workflows/m68k.yml

```diff
@@ -88,6 +91,19 @@ jobs:
         sudo mount debian-m68k.img vm
         sudo cp $(which qemu-m68k-static) vm/usr/bin/
 
+    - name: Prepare for build with target as JSON spec
+      run: |
+        ./y.sh prepare --only-libcore --cross
+
+    - name: Build sysroot with target as JSON spec
+      run: |
+        ./y.sh build --target-triple m68k-unknown-linux-gnu --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
+
+    - name: Build with target as JSON spec
+      run: |
+        ./y.sh cargo build --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
```

Comment:
I believe the issue is because you're trying to build the GCC codegen itself for m68k.
Please try to compile [this test project](https://github.com/antoyo/rustc_codegen_gcc/tree/master/tests/hello-world) instead.

Reply:
Fixed